deploy/deploy.go

```diff
@@ -219,6 +219,8 @@ type KindSpec struct {
 	Kubecfg                  string            `yaml:"kubecfg"`
 	GoogleArtifactRegistries []string          `yaml:"googleArtifactRegistries"`
 	ContainerImages          map[string]string `yaml:"containerImages"`
+	KindConfigFile           string            `yaml:"config"`
+	AdditionalManifests      []string          `yaml:"additionalManifests"`
```

Comment:
this field could technically be used to apply manifests that are not kind related, but not sure of a better place to put this field

Reply:
this area it is really "part of" kind deployment so i think it stands alone
if we want to make a post cluster bringup hook we could do that but for now i think this is as good a place of as any